src/System.Memory/src/System/Buffers/MemoryHandle.cs

```diff
@@ -20,19 +20,24 @@ public unsafe struct MemoryHandle : IDisposable
         /// Creates a new memory handle for the memory.
         /// </summary>
         /// <param name="retainable">reference to manually managed object</param>
-        /// <param name="pinnedPointer">pointer to the buffer, or null if the buffer is not pinned</param>
+        /// <param name="pointer">pointer to the buffer, or null if the buffer is not pinned</param>
```

Comment:
Is the "is not pinned" part correct?  Part of the reasoning behind removing "Pinned" as a prefix I thought was that this may just be native memory for which there is no concept of pinning.

Reply:
I would just say "pointer to memory, or null if a pointer was not provided when the handle was created."